src/image/labelingDebug.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Helper for the LabelingFilter, provides a debug display for diameters and
+ * contours.
+ *
+ * Temporary until a proper way of rendering them is added.
+ */
+export class LabelingDebug {
+
+  /**
+   * Cleans buffer of debug lines before calculations.
+   *
+   * @param {TypedArray} imageBuffer The image buffer to clean.
+   */
+  cleanBuffer(imageBuffer) {
+    for (let i = 0; i < imageBuffer.length; i++) {
+      if (imageBuffer[i] >= 128) {
+        imageBuffer[i] = imageBuffer[i] - 128;
+      }
+    }
+  }
+
+  /**
+   * Convert a slice local world coordinate to a slice local offset value.
+   *
+   * @param {object} point Point on the slice, scaled.
```

Comment:
there is a Point2D class in the math folder, could you use it?

Reply:
updated